src/main/java/dukeclasses/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package dukeclasses;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * dukeClasses.Deadline class is a child class of dukeClasses.Task that has a by,
+ * variable which tells the user when the deadline is.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate by;
+
+    public Deadline(String description, LocalDate by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
+     * returns a String containing description of object.
+     *
```

Comment:
I think this is an additional line that can be deleted

Reply:
Alright thank you! I'll take note of it